sprockets/mixins/metrics/testing.py

```diff
@@ -36,7 +33,7 @@ class FakeStatsdServer(object):
     def __init__(self, iol):
         self.socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM,
                                     socket.IPPROTO_UDP)
-        self.socket.bind(('127.0.0.1', 0))
+        self.socket.bind(('127.0.0.1', 8125))
```

Comment:
I was explicitly generating an ephemeral port here so that tests don't fail when run on a machine with statsd running.